Tests/DatadogTests/Datadog/Mocks/RUMDataModelMocks.swift

```diff
@@ -0,0 +1,199 @@
+                ssl: .init(duration: .mockRandom(), start: .mockRandom()),
+                statusCode: .mockRandom(),
+                type: [.xhr, .fetch, .image].randomElement()!,
+                url: .mockRandom()
+            ),
+            service: .mockRandom(),
+            session: .init(
+                id: .mockRandom(),
+                type: .user
+            ),
+            usr: .mockRandom(),
+            view: .init(
+                id: .mockRandom(),
+                referrer: .mockRandom(),
+                url: .mockRandom()
+            )
+        )
+    }
+}
+
+extension RUMActionEvent {
+    static func mockRandom() -> RUMActionEvent {
+        return RUMActionEvent(
+            dd: .init(),
+            action: .init(
+                crash: .init(count: .mockRandom()),
+                error: .init(count: .mockRandom()),
+                id: .mockRandom(),
+                loadingTime: .mockRandom(),
+                longTask: .init(count: .mockRandom()),
+                resource: .init(count: .mockRandom()),
+                target: .init(name: .mockRandom()),
+                type: [.tap, .swipe, .scroll].randomElement()!
+            ),
+            application: .init(id: .mockRandom()),
+            connectivity: .mockRandom(),
+            date: .mockRandom(),
+            service: .mockRandom(),
+            session: .init(
+                id: .mockRandom(),
+                type: .user
+            ),
+            usr: .mockRandom(),
+            view: .init(
+                id: .mockRandom(),
+                referrer: .mockRandom(),
+                url: .mockRandom()
+            )
+        )
+    }
+}
+
+extension RUMErrorEvent {
+    static func mockRandom() -> RUMErrorEvent {
+        return RUMErrorEvent(
+            dd: .init(),
+            action: .init(id: .mockRandom()),
+            application: .init(id: .mockRandom()),
+            connectivity: .mockRandom(),
+            date: .mockRandom(),
+            error: .init(
+                isCrash: .random(),
+                message: .mockRandom(),
+                resource: .init(
+                    method: .mockRandom(),
+                    provider: .init(
+                        domain: .mockRandom(),
+                        name: .mockRandom(),
+                        type: Bool.random() ? .firstParty : nil
+                    ),
+                    statusCode: .mockRandom(),
+                    url: .mockRandom()
+                ),
+                source: [.source, .network, .custom].randomElement()!,
+                stack: .mockRandom()
+            ),
+            service: .mockRandom(),
+            session: .init(
+                id: .mockRandom(),
+                type: .user
+            ),
+            usr: .mockRandom(),
+            view: .init(
+                id: .mockRandom(),
+                referrer: .mockRandom(),
+                url: .mockRandom()
+            )
+        )
+    }
+}
```

Comment:
can we use model generator to create these mocks too?

Reply:
I was thinking of this, but it took me `~10 min` to do it by hand, while adding this in `rum-models-generator` would be at least few hours. Let's see how difficult would it be to maintain this hand-made mocks, if too expensive, we can invest our time in `rum-models-generator` 👌.